game/Switch_Miidatafile.ksy

```diff
@@ -0,0 +1,167 @@
+meta:
+  id: miidata_swi
+  endian: le
+seq:
+  - id: unknown_data
+    type: u1
+    repeat: expr
+    repeat-expr: 16
+    doc: Currently unknown data.
+  - id: mii_name
+    type: str
+    size: 20
+    encoding: utf-16le
+    doc: Mii name. Can be up to 10 characters long. Different from the Mii name that appears in Super Smash Bros. Ultimate - in game, this is never seen.
+  - id: unknown_buffer
+    type: u1
+    repeat: expr
+    repeat-expr: 3

```

Comment:
If you're relatively sure that these bytes should always be zero, you can use `contents` to check that:

```suggestion
    contents: [0, 0, 0]
```

Reply:
> If you're relatively sure that these bytes should always be zero, you can use `contents` to check that

Yes, _"if you're relatively sure"_, but I don't think we can assume anything about _unknown data_.